ime/app/src/main/java/com/anysoftkeyboard/keyboards/AnyKeyboard.java

```diff
@@ -833,12 +833,7 @@ public AnyKey(
             // if the shift-character is a symbol, we only show it if the SHIFT is pressed,
             // not if the shift is active.
             // Note: above doesn't work, it only shows the symbols if SHIFT is being held.
-            mShiftCodesAlways =
-                    mShiftedCodes.length == 0
-                            || Character.isLetter(mShiftedCodes[0])
-                            || Character.getType(mShiftedCodes[0]) == Character.NON_SPACING_MARK
-                            || Character.getType(mShiftedCodes[0])
-                                    == Character.COMBINING_SPACING_MARK;
+            mShiftCodesAlways = true;
```

Comment:
does this mean that `mShiftCodesAlways` is `true` is all cases? Can we remove it and all the branches that use it?

Reply:
Sounds about right.